tests/test_ext_intersphinx.py

```diff
@@ -12,13 +12,18 @@
 import unittest
 
 from docutils import nodes
+from textwrap import dedent
 import mock
+import pytest
+import requests
+from io import BytesIO
```

Comment:
Hmm... I don't know that.
Okay, let's go with `io.BytesIO`. Let's consider again if some trouble happeded.